plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -231,19 +231,21 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
 
       override fun onAnimationStart(animation: Animator?) {
```

Comment:
What if we checked if the `animation` that's being started here is part of the high-level animator instead of adding the `started` flag? Instead of mixing low-level and high-level animators together, we could have 2 logical branches for them.

Not a blocker, but might be something to consider for a refactor.

Reply:
Nice suggestion, indeed that logic is becoming more and more complicated. It would be nice to differentiate those animators in some better way. I'll think about it and create some follow-up ticket shortly.

Reply:
Taken another look - could not see an easy way right now to differentiate those.. Basically it does not matter much for camera animation engine if animation started is part of high-level animation or stand alone low-level animation. Guess current approach is fine for now, I will put this thing on my radar and perhaps try to prototype some easier way and then wrap it to a concrete ticket.